src/backend/marsha/markdown/serializers.py

```diff
@@ -1,12 +1,152 @@
 """Structure of Markdown related models API responses with Django Rest Framework serializers."""
+import mimetypes
+from os.path import splitext
+
+from django.conf import settings
 from django.urls import reverse
 
 from rest_framework import serializers
 
+from marsha.core.serializers import (
+    InitiateUploadSerializer,
+    TimestampField,
+    UploadableFileWithExtensionSerializerMixin,
+    get_resource_cloudfront_url_params,
+)
 from marsha.core.serializers.playlist import PlaylistLiteSerializer
+from marsha.core.utils import cloudfront_utils, time_utils
 from marsha.core.utils.url_utils import build_absolute_uri_behind_proxy
 
-from .models import MarkdownDocument
+from .models import MarkdownDocument, MarkdownImage
+
+
+class ReadOnlyModelSerializer(serializers.ModelSerializer):
```

Comment:
What about moving this to `core` to allow to use it in all marsha's applications ?